### DISCUSSION TITLE
About random gamma

### DISCUSSION THREAD

**Discussion by alonfnt** [deepmind/dm_pix#43] [General]
Is there a reason `adjust_gamma` is missing its random equivalent as in the case of `adjust_contrast`, `adjust_brightness`, `adjust_hue` and `adjust_saturation`?

**Comment by claudiofantacci** [comment]
Hi @alonfnt 😄 
Not really! If you're keen to contribute with an implementation, feel free to send a PR 🚀

**Comment by alonfnt** [comment]
Great! I had doubts on whether `dm_pix` was supposed to be a one-to-one JAX equivalent to `tf.image`. 

Good to know that it can go beyond it!

**Comment by claudiofantacci** [comment]
The library grows with users contributions, so it might be that some more common functions are there already, while other will take more time to release 😄
